copyall.sh

```diff
@@ -900,6 +911,11 @@ echo "copying root@${centos75x64update7}:/root/rpmbuild/RPMS/x86_64/* LISISO/Ora
 scp -r root@${centos75x64update7}:/root/rpmbuild/RPMS/x86_64/* LISISO/Oracle75/update7/
 scp -r root@${centos75x64update7}:/root/rpmbuild/SRPMS/* LISISO/Oracle75/update7/
 
+echo "copying root@${centos76x64}:/root/rpmbuild/RPMS/x86_64/* LISISO/Oracle76"
+#copying file from Oracle7.6
+scp -r root@${centos76x64}:/root/rpmbuild/RPMS/x86_64/* LISISO/Oracle76/
+scp -r root@${centos76x64}:/root/rpmbuild/SRPMS/* LISISO/Oracle75/
```

Comment:
Modify the Version from Oracle75 to Oracle76.

Reply:
corrected the copy path